pkg/sms/twilio_test.go

```diff
@@ -88,8 +91,23 @@ func TestTwilio_SendSMS(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			err = twilio.SendSMS(ctx, tc.to, "testing 123")
-			if (err != nil) != tc.err {
+			// Twilio is pretty flaky, retry if it failed unexpectedly
+			b, err := retry.NewConstant(100 * time.Millisecond)
+			if err != nil {
+				t.Fatalf("failed to configure backoff: %v", err)
+			}
+			b = retry.WithMaxRetries(3, b)
+
+			if err := retry.Do(ctx, b, func(_ context.Context) error {
+				err = twilio.SendSMS(ctx, tc.to, "testing 123")
+				if err != nil && err.Error() == "The 'To' number +15005550006 is not a valid phone number" {
```

Comment:
Is there something to compare against that isn't a hardcoded error string?

Reply:
As a custom error wrapped based off of a REST call response body, I'm afraid that `errors.Is` is not applicable in this case.

Reply:
even strings.Contains(err.Error(), "not a valid phone number") would make me feel a little better?

Reply:
I see your point, unfortunately that probably won't work. See https://github.com/google/exposure-notifications-verification-server/blob/85a4fe8e9339914360d0099c9ca002fa5806d52d/pkg/sms/twilio_test.go#L37

That `15005550001` was designed to get this error, which is not a case that should be retried

Reply:
ah. Maybe const the magic number with a comment?

Reply:
Make sense, done